src/TaskManager.Core/Services/EmployeeService.cs

```diff
@@ -0,0 +1,26 @@
+﻿using TaskManager.Core.Interfaces.Data;
+using TaskManager.Core.Interfaces.Services;
+using TaskManager.Core.Models;
+
+namespace TaskManager.Core.Services;
+
+public class EmployeeService : Service<Employee>, IEmployeeService 
+{
+    private readonly IRepository<Employee> _employeeRepository;
+
+    public EmployeeService(IRepository<Employee> employeeRepository) : base(employeeRepository)
+    {
+        _employeeRepository = employeeRepository ?? throw new ArgumentNullException(nameof(employeeRepository));
+    }
+
+    public Task<List<Employee>> GetEmployeesByProjectIdAsync(Guid projectId, CancellationToken cancellationToken) 
+    {
+        return _employeeRepository.FindAsync(t => t.Projects.Any(t => t.Id == projectId), cancellationToken); 
```

Comment:
косметический коммент чисто от меня: мне не нравится, когда в лямбдах позволяют себе называть переменные отпизды🥲 Почему переменные не в люмбдах нормально называем, а в них можно просто букву поставить? Могу еще принять, когда в цепочке только одна лямюда и буква это эта хотя бы первая из реального названия, но все равно хуйня, по-моему. 
Имхо не сложно написать целое слово. Что здесь значит t? Почему t? Почему две разных переменных, одна означает employee, а другая project и обе называются t, что перекрывает друг друга?
```suggestion
        return _employeeRepository.FindAsync(employee => employee.Projects.Any(project => project.Id == projectId), cancellationToken); 
```